projects/cn0506_mii/a10soc/system_project.tcl

```diff
@@ -0,0 +1,118 @@
+
+source ../../scripts/adi_env.tcl
+source ../../scripts/adi_project_intel.tcl
+
+adi_project cn0506_a10soc
+
+source $ad_hdl_dir/projects/common/a10soc/a10soc_system_assign.tcl
+
+# Note: This projects requires a hardware rework to function correctly.
+# The rework connects FMC header pins directly to the FPGA so that they can be
+# accessed by the fabric.
+#
+# Changes required (D08-FMC):
```

Comment:
Do we need the changes?

Reply:
The RX_ER_A signal is connected to the D08/LA01 pin on the FMC connector, so if we want to use RX_ER_A, yes, the change is required.